examples/vlue_evaluation/README.md

```diff
@@ -0,0 +1,111 @@
+------------------------------------------ | ----- | ----- | -----
+S2                                         | 0.903 | 0.167 | 1.000
+Grobid                                     | 0.782 | 0.000 | 1.000
+VILA (IVILA-docbank-layoutlm-base-uncased) | 0.614 | 0.000 | 0.992
+
+### Abstract
+
+Model                                      | Mean  | Min   | Max
+------------------------------------------ | ----- | ----- | -----
+S2                                         | 0.787 | 0.000 | 1.000
+Grobid                                     | 0.780 | 0.000 | 1.000
+VILA (IVILA-docbank-layoutlm-base-uncased) | 0.671 | 0.000 | 0.989
+
+These scores are based on 18 of the documents excluding the following SHAs which fail for VILA:
+
+```
+396fb2b6ec96ff74e22ddd2484a9728257cccfbf
+3ef6e51baee01b4c90c188a964f2298b7c309b07
+4277d1ec41d88d595a0d80e4ab4146d8c2db2539
+564a73c07436e1bd75e31b54825d2ba8e4fb68b7
+```
+
+## Dataset labels
+
+Labels used in the evaluation here have the following format:
+
+```json
+{
+  "id": "semantic-scholar-paper-hash",
+  "url": "URL to find paper on semanticscholar.org",
+  "title": "Paper title in UTF-8",
+  "abstract": "Paper abstract in UTF-8"
+}
+```
+
+The labels are hand-curated and attempt to be the best possible output for a paper given just the PDF and no external metadata about the paper, authors, etc. This generally means that:
+
+1. Characters use a Unicode representation when possible. An example of this is the superscript "+" used when referring to a positively charged ion. For example "Na+" vs. "Na⁺". The latter is used when it visually matches what is shown on the PDF regardless of any underlying character stream.
+
+2. When multiple languages are available in the paper the English language summary is provided only. A future version of this dataset may provide additional languages.
+
+3. Words are dehyphenated. The task here is a reconstruction of the full text without visual breaks (like multi-line hyphenation). In other words, the output of a prediction should be directly consumable by another system that expects fully reconstructed text (e.g., a screen reader, classifier, etc.).
+
+4. Multi-paragraph abstracts are joined by a single newline character between paragraphs.
+
+## Obtaining the labels
+
+Currently, labels are kept on (AI2-private) AWS:
+
+```bash
+aws s3 cp s3://ai2-s2-russellr/datasets/mmda/vlue-curated-labels.json
+```
+
+This location will change as the dataset is completed and more documents are added. Currently, only 22 documents are available.
+
+## Metrics
+
+Currently there is a single metric provided:
+
+```python
+def score(label: str, prediction: str) -> float:
+  return 1 - (levenshtein(label, prediction) / max(len(label), len(prediction)))
+```
+
+See this [Wikipedia article](https://en.wikipedia.org/wiki/Levenshtein_distance) for more information.
+
+## Running the evaluation
+
+For a single VILA model run use the following (using `main.py` in this folder):
+
+```bash
+python main.py \
```

Comment:
can this script output a `preds.csv` in addition to printing the summary stats above?  we'll likely want to analyze the individual predictions

Reply:
Yep.